src/current/_includes/v23.1/misc/tooling.md

```diff
@@ -66,7 +66,7 @@ For a list of tools supported by the CockroachDB community, see [Third-Party Too
 
 | IDE | Latest tested version | Support level | Tutorial |
 |-----+-----------------------+---------------+----------|
-| [DataGrip](https://www.jetbrains.com/datagrip/) | 2021.1 | Full | N/A
+| [DataGrip](https://www.jetbrains.com/datagrip/) | 2024.1 | Full | N/A
 | [IntelliJ IDEA](https://www.jetbrains.com/idea/) | 2021.1 | Full | [Use IntelliJ IDEA with CockroachDB]({% link {{ page.version.version }}/intellij-idea.md %})
```

Comment:
We can also update the IntelliJ IDEA version? cc @dikshant

Reply:
Yep they are effectively the same!

Reply:
ok great, thanks - i updated them both to 2024.1